topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md

```diff
@@ -0,0 +1,159 @@
+
+By default the `igGridExcelExporter` will copy styles from the header row and the first two data rows from `igGrid` to apply the same to the Worksheet table. If `gridStyling` is set to `"none"`, then a predefined Excel table style will be applied to the worksheet. Table styles follow the [ECMA specification for Office Open XML](http://www.ecma-international.org/news/TC45_current_work/TC45_available_docs.htm) (see section on Table Style Definitions). The predefined style may be set using the `tableStyle` and the available choices are divided into three categories:
+`TableStyleMedium[1-28]`, `TableStyleLight[1-21]` and `TableStyleDark[1-11]`. 
+The range in brackets is the available theme number that may be chosen:
+
+```javascript
+$.ig.GridExcelExporter.exportGrid($(".selector"), {
+    gridStyling: "none",
+    //Setting the table style to 18th theme from TableStyleMedium category
+    tableStyle: "TableStyleMedium18"
+ });
+ ```
+
+### <a id="callbacks"></a>Attach to callbacks (events)
+
+A number of callbacks (events) are exposed that are called during the exporting process and provide entry points to customize or end the exporting process. Exporting/ed callbacks that are available for cells and rows are suitable to apply formulas and conditional formatting, read/write values from the Worksheet object, etc. There are also the `exportStarting`, `exportEnding` and `error` and `success` callbacks, which are suitable to verify anything before exporting or before saving the file locally, or to display a modal overlay over the `igGrid` while exporting, as shown in the [Display an overlay while exporting](#exporting_overlay) section.
+
+
+```javascript
+$.ig.GridExcelExporter.exportGrid($("#grid1"), {},
+{
+	cellExported: function (e, args) {
+		if (args.columnKey == 'Company') {
+			var xlRow = args.xlRow;
+            // apply formula to a cell
+			xlRow.cells(args.columnIndex).applyFormula('=HYPERLINK("' + args.cellValue + '")');
+		},
+		if (args.columnKey == "Age" && args.cellValue > 43) {
+            // format the font and the cell fill color
+			args.xlRow.getCellFormat(args.columnIndex).font().bold(1);
+            args.xlRow.getCellFormat(args.columnIndex).fill($.ig.excel.CellFill.createLinearGradientFill(45, '#FF0000', '#00FFFF'));
+		}
+	},
+    exportEnding: function (e, args) {
+        // format a date value to be displayed as 19/Mar/2017
+        args.worksheet.columns().item(3).cellFormat().formatString("dd/MMM/YYYY");
+    },
+});
+ ```
+
+### <a id="exporting_overlay"></a>Display an overlay while exporting
+
+During exporting the `igGrid` UI is not available for interactions so you may want to place a modal overlay above. It will show in the beginning of exporting and hide in the end. To implement this the `exportStarting` and `success` callbacks are used:
+
+```javascript
+$.ig.GridExcelExporter.exportGrid($("#grid1"), {},
+{
+    exportStarting: function (e, args) {
+        showOverlay(args.grid, exportingOverlay);
```

Comment:
As a reader I'm interested what is the definition of the `showOverlay` and `hideOverlay` functions. Please add their definitions to the code sample or better add a live sample.